aws/aws-toolkit-vscode
# Tidy CDK Commands

## Description

This change cleans up some of the experience around commands and the Command Palette for consistency within the toolkit.

* The Refresh commands were renamed from "Refresh" to "Refresh AWS Explorer" and "Refresh CDK Explorer" to disambiguate the commands
  * They show as tooltips:
![image](https://user-images.githubusercontent.com/39839589/69886265-f3ec5400-1295-11ea-9ce1-20435373b9eb.png)

* The "Refresh CDK Explorer" command has been removed from the Command Palette (it is accessible from the CDK Explorer, and this parallels the AWS Explorer's refresh functionality)
* The "Provide Feedback" command that is available from the CDK Explorer has been removed from the Command Palette to avoid confusion with the "Report an Issue" command
* The "Refresh CDK Explorer" command has been added to the extension activation list in order to prevent a "command not found" error message if the command is clicked before the extension fully activates

## Testing

Manually tested in-toolkit:
* Verified that the AWS Explorer and CDK Explorer's refresh commands have the updated tooltip
* Verified that the Refresh command for CDK Explorer is no longer in the Command Palette
* Verified that the Provide Feedback command for CDK Explorer is no longer in the Command Palette
* Clicked the "Refresh CDK Explorer" button as soon as VS Code loads and before the toolkit is activated, to ensure the error message no longer occurs

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
